extensions/machine-learning/src/common/utils.ts

```diff
@@ -44,6 +44,15 @@ export async function exists(path: string): Promise<boolean> {
 	return promisify(fs.exists)(path);
 }
 
+export async function isDirectory(path: string): Promise<boolean> {
+	try {
+		const stat = await fs.promises.lstat(path);
```

Comment:
I use fs itself too so I wanted to make it clear. otherwise it'll look like promises.promises 

---
In reply to: [425278988](https://github.com/microsoft/azuredatastudio/pull/10377#discussion_r425278988) [](ancestors = 425278988)